lib/book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr_reader :author_first_name, :author_last_name, :title, :publication_date
+
+  def initialize(book_info)
+    @author_first_name = book_info[:author_first_name]
+    @author_last_name = book_info[:author_last_name]
+    @title = book_info[:title]
+    @publication_date = book_info[:publication_date]
+  end
+
+  def author
+    [@author_first_name, @author_last_name].join(" ")
+  end
+
+  def publication_year
+    @publication_date.to_s.inspect[10..13]
```

Comment:
This is neat. A few things: 1) pub_date is already a string, 2) what if I submitted a date like "Feb 12, 1970", now this is broken because `[10..13]` won't always work. We didn't specifically ask to handle this edge case, but it'd be better to use `[-4..-1]` to grab the last 4 characters of the string.